sql-parser/src/main/java/io/crate/sql/tree/Assignment.java

```diff
@@ -25,39 +25,67 @@
 import com.google.common.base.Objects;
 import com.google.common.base.Preconditions;
 
+import java.util.Collections;
+import java.util.List;
+
 public class Assignment extends Node {
 
     private final Expression columnName;
-    private final Expression expression;
-
+    private final List<Expression> expressions;
+
+    public Assignment(Expression columnName, List<Expression> expressions) {
+        /**
+         * Constructor for SET SESSION/LOCAL statements
+         * one or more expressions are allowed on the right side of the assignment
```

Comment:
imho the comments here could be proper public javadocs (placed above the constructor)